sync-service/Dockerfile

```diff
@@ -13,6 +13,5 @@ FROM openjdk:11-jdk-slim
 COPY --from=build /workspace/target/*.jar app.jar
 EXPOSE 8080
 COPY start.sh /start.sh
-RUN chmod +x /start.sh
```

Comment:
@UncleSamSwiss removed as it is not needed as long as the file is executable in git